packages/botonic-react/src/webchat/components/emoji-picker.jsx

```diff
@@ -64,7 +64,13 @@ export const OpenedEmojiPicker = props => {
     <div ref={ref}>
       {isComponentVisible && (
         <Container role={ROLES.EMOJI_PICKER}>
-          <Picker onEmojiClick={props.onEmojiClick} disableAutoFocus={true} />
+          <Picker
+            width='100%'
+            height='19rem'
```

Comment:
if we use rem and the user has a very large font size in the default browser, this will be larger than expected.
I think it is better to fix this with px

Reply:
let's change it in the next PR 👍